cmd/bosun/sched/check.go

```diff
@@ -111,7 +111,20 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 		if event.Unevaluated {
 			continue
 		}
+		prev := state.Last()
 		state.Append(event)
+		if prev.IncidentId != 0 {
+			//if last event has incident id and is not closed, we continue it.
```

Comment:
comments have a space after `//`, and start with caps: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences